da/src/main/java/org/jboss/bacon/da/DAReportsCli.java

```diff
@@ -0,0 +1,223 @@
+    public static class AlignReport extends JSONCommandHandler implements Callable<Integer> {
+
+        @CommandLine.Option(
+                names = "--repositories",
+                description = "Aditional maven repositories required by the analysed project.")
+        private List<String> repositories = new ArrayList<>();
+
+        @CommandLine.Parameters(description = "Scm url of project repository to analyze.")
+        private String scm;
+
+        @CommandLine.Parameters(description = "Revision to analyze.")
+        private String revision;
+
+        @CommandLine.Parameters(description = "Path to pom.xml file.")
+        private String pomPath;
+
+        @Override
+        public Integer call() throws Exception {
+            AlignReportRequest request = new AlignReportRequest();
+            request.setScmUrl(scm);
+            request.setRevision(revision);
+            request.setPomPath(pomPath);
+            request.setAdditionalRepos(repositories);
+
+            ReportsApi reportsApi = DaHelper.createReportsApi();
+            try {
+                org.jboss.da.reports.model.response.striped.AlignReport result = WLStripper
+                        .strip(reportsApi.alignReport(request));
+                ObjectHelper.print(false, result);
+            } catch (IOException e) {
+                throw new RuntimeException(e);
+            }
+            return 0;
+        }
+    }
+
+    @CommandLine.Command(name = "built", description = "Get built artifacts for project specified in a repository URL.")
+    public static class BuiltReport extends JSONCommandHandler implements Callable<Integer> {
+
+        @CommandLine.Option(
+                names = "--repositories",
+                description = "Aditional maven repositories required by the analysed project.")
+        private List<String> repositories = new ArrayList<>();
+
+        @CommandLine.Parameters(description = "Scm url of project repository to analyze.")
+        private String scm;
+
+        @CommandLine.Parameters(description = "Revision to analyze.")
+        private String revision;
+
+        @CommandLine.Parameters(description = "Path to pom.xml file.")
+        private String pomPath;
+
+        @Override
+        public Integer call() throws Exception {
+            BuiltReportRequest request = new BuiltReportRequest();
+            request.setScmUrl(scm);
+            request.setRevision(revision);
+            request.setPomPath(pomPath);
+            request.setAdditionalRepos(repositories);
+
+            ReportsApi reportsApi = DaHelper.createReportsApi();
+            try {
+                Set<org.jboss.da.reports.model.response.BuiltReport> result = reportsApi.builtReport(request);
+                ObjectHelper.print(false, result);
+            } catch (IOException e) {
+                throw new RuntimeException(e);
+            }
+            return 0;
+        }
+    }
+
+    @CommandLine.Command(
+            name = "lookup-versions-npm",
```

Comment:
@janinko we wanted to keep lookup for npm versions right? this one is not in new lookup endpoint

Reply:
yup